unleash-yggdrasil/src/lib.rs

```diff
@@ -673,11 +675,11 @@ mod test {
     use serde::Deserialize;
     use std::{collections::HashMap, fs};
     use test_case::test_case;
-    use unleash_types::client_features::{ClientFeatures, FeatureDependency, Override};
+    use unleash_types::client_features::{ClientFeatures, FeatureDependency, Override, Payload};
```

Comment:
This and everything below in this file is just a consequence of having `feature_enabled` now properly camelCased on serialise/deserialise.